updatehub/src/main.rs

```diff
@@ -114,7 +114,31 @@ async fn client_main(cmd: ClientCommands) -> updatehub::Result<()> {
 
     match cmd {
         ClientCommands::Info(_) => println!("{:#?}", client.info().await),
-        ClientCommands::Log(_) => println!("{:#?}", client.log().await),
+        ClientCommands::Log(_) => {
+            for entry in client.log().await?.into_iter() {
+                let level = match entry.level {
+                    sdk::api::log::Level::Critical => "CRIT",
+                    sdk::api::log::Level::Error => "ERRO",
+                    sdk::api::log::Level::Warning => "WARN",
+                    sdk::api::log::Level::Info => "INFO",
+                    sdk::api::log::Level::Debug => "DEBG",
+                    sdk::api::log::Level::Trace => "TRCE",
+                };
+
+                println!(
+                    "[{timestamp}] {level} {msg}",
+                    timestamp = entry
+                        .time
+                        .chars()
+                        .take(19)
+                        .collect::<Vec<_>>()
+                        .into_iter()
+                        .collect::<String>(),
+                    level = level,
+                    msg = entry.message
```

Comment:
```diff
diff --git a/updatehub/src/main.rs b/updatehub/src/main.rs
index 340da91..8849337 100644
--- a/updatehub/src/main.rs
+++ b/updatehub/src/main.rs
@@ -126,7 +126,7 @@ async fn client_main(cmd: ClientCommands) -> updatehub::Result<()> {
                 };
 
                 println!(
-                    "[{timestamp}] {level} {msg}",
+                    "{timestamp} {level} {msg}",
                     timestamp = entry
                         .time
                         .chars()
diff --git a/updatehub/tests/common.rs b/updatehub/tests/common.rs
index ed96ad0..145aa0c 100644
--- a/updatehub/tests/common.rs
+++ b/updatehub/tests/common.rs
@@ -312,9 +312,6 @@ pub fn remove_carriage_newline_characters(s: String) -> String {
 }
 
 pub fn format_output_client_log(s: String) -> String {
-    let s = s.replace("[", "");
-    let s = s.replace("]", "");
-
     let date_re = Regex::new(r"(\d{4})-(\d{2})-(\d{2}) (\d{2}):(\d{2}):(\d{2})")
         .expect("fail to compile the date regexp");
     let s = date_re.replace_all(&s, "<timestamp>");
```

Reply:
The original log also has the `[...]` around the timestamp?

Reply:
On me review on this item there's a sample on the original timestamp format.